sdk/src/test/java/com/hedera/hashgraph/sdk/ExchangeRateSetTest.java

```diff
@@ -0,0 +1,32 @@
+package com.hedera.hashgraph.sdk;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import org.bouncycastle.util.encoders.Hex;
+import org.junit.jupiter.api.Test;
+
+import org.threeten.bp.Instant;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ExchangeRateSetTest {
+    private static final String exchangeRateSetHex = "0a1008b0ea0110b6b4231a0608f0bade9006121008b0ea01108cef231a060880d7de9006";
```

Comment:
Feel like we should use a snapshot test here instead.

Reply:
Not sure how to go about this, might need some assistance.